#! /bin/sh /usr/share/dpatch/dpatch-run
## libcollectdclient_static_sstrerror.dpatch by Sebastian Harl <sh@tokkee.org>
##
## DP: libcollectdclient/client.c: Made sstrerror() static.
## DP:
## DP: This is a private helper function only.

@DPATCH@

diff a/src/libcollectdclient/client.c b/src/libcollectdclient/client.c
--- a/src/libcollectdclient/client.c
+++ b/src/libcollectdclient/client.c
@@ -121,7 +121,7 @@ typedef struct lcc_response_s lcc_response_t;
 /* Even though Posix requires "strerror_r" to return an "int",
  * some systems (e.g. the GNU libc) return a "char *" _and_
  * ignore the second argument ... -tokkee */
-char *sstrerror (int errnum, char *buf, size_t buflen)
+static char *sstrerror (int errnum, char *buf, size_t buflen)
 {
   buf[0] = 0;
 
